### PR TITLE
K8 tests

### PR DESCRIPTION
**Pull request by shrutir25**
**What this PR does / why we need it**: It adds a sleep time of 5 mins before checking count of nodes in Kubernetes clusters

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
It addresses the Nodes Not Ready issue for #699 

**Special notes for your reviewer**:

**Release note**:

```release-note
```

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/735)

### CODE REVIEW

**Comment by shrutir25** [comment]
@dmitsh could you review this ?

File: test/acs-engine-test/main.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func (m *TestManager) Run() error {
 func (m *TestManager) testRun(d Deployment, index, attempt int, timeout time.Duration) bool {
 	name := strings.TrimSuffix(d.ClusterDefinition, filepath.Ext(d.ClusterDefinition))
 	instanceName := fmt.Sprintf("acse-%d-%s-%s-%d-%d", rand.Intn(0x0ffffff), d.Location, os.Getenv("BUILD_NUMBER"), index, attempt)
-	resourceGroup := fmt.Sprintf("x-%s-%s-%s-%d-%d", strings.Replace(name, "/", "-", -1), d.Location, os.Getenv("BUILD_NUMBER"), index, attempt)
+	resourceGroup := fmt.Sprintf("y-%s-%s-%s-%d-%d", strings.Replace(name, "/", "-", -1), d.Location, os.Getenv("BUILD_NUMBER"), index, attempt)
```
Comment (dmitsh) [inline test/acs-engine-test/main.go]: this is your local change. No need to submit

File: test/cluster-tests/kubernetes/test.sh [hunk 2]
```diff
@@ -55,18 +55,23 @@ if [[ "${TEST_ACR}" == "y" ]]; then
 	fi
 fi
 
+sleep 5m
+log "Checking for Node Count"
 ###### Check node count
 function check_node_count() {
-  log "Checking node count"
```
Comment (dmitsh) [inline test/cluster-tests/kubernetes/test.sh]: please keep log

File: test/acs-engine-test/acse-k8s-test.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (dmitsh) [inline test/acs-engine-test/acse-k8s-test.json]: this is local to your branch

File: test/cluster-tests/kubernetes/test.sh (hunk 2 above)
Comment (shrutir25) [inline test/cluster-tests/kubernetes/test.sh]: I had moved it outside the function. I will move it back inside

**Comment by shrutir25** [comment]
closing PR as it has lot of previous uneccessary commits.

**Pull request closed without merging**